Update glass_hover_effect.dart

the-lastbit opened the pull request:
Add fix for rounded corners in the BackDropFilter widget

the-lastbit committed:
Update glass_hover_effect.dart

Add fix for rounded corners in the BackDropFilter

md-weber commented:
Hi there, thanks for your contribution!
I tested it quickly and saw that you fixed on thing but you introduced another bug. Now the text is always visible on the screen.

![image](https://github.com/Flutter-Explained/hover_effects/assets/8026644/c8d3965f-e5ac-4f26-8533-e873a7327f51)

md-weber reviewed (changes requested):
Hi there, thanks for your contribution!
I tested it quickly and saw that you fixed on thing but you introduced another bug. Now the text is always visible on the screen.

![image](https://github.com/Flutter-Explained/hover_effects/assets/8026644/c8d3965f-e5ac-4f26-8533-e873a7327f51)

the-lastbit committed:
Update glass_hover_effect.dart

Sorry man, I didn't see the message about my mistake, forgot to wrap the SizedBox with another ClipRect because I was debugging in a phone and the narrow screen didn't show the text in the right side 😅, But now it works.
